app/src/main/java/com/imashnake/animite/features/searchbar/Search.kt

```diff
@@ -213,7 +245,14 @@ fun SearchList(
     if (!searchList.isNullOrEmpty()) {
         LazyColumn(
             modifier = modifier,
-            contentPadding = PaddingValues(16.dp)
+            contentPadding = PaddingValues(
+                start = dimensionResource(Res.dimen.medium_padding),
+                end = dimensionResource(Res.dimen.medium_padding),
+                bottom = dimensionResource(Res.dimen.search_bar_height)
+                        + dimensionResource(Res.dimen.large_padding)
+                        + dimensionResource(Res.dimen.large_padding),
+                top = WindowInsets.statusBars.asPaddingValues().calculateTopPadding()
+            )
```

Comment:
What on earth is going on with the padding here

Reply:
![Group 6(2)](https://user-images.githubusercontent.com/56754616/202067268-877721b5-8219-47df-af17-c0c7e2a12dba.png)